doc/guides/getting_started/src/e2e_regression_setup.md

```diff
@@ -0,0 +1,55 @@
+# ROM E2E Regressions setup
+This guide will help you to setup an environment to run the ROM E2E tests in an Ubuntu 20.04 machine.
+
+## Introduction
+The [ROM](https://opentitan.org/book/doc/security/specs/secure_boot/index.html?highlight=ROM#rom-rom) is first boot stage of secure boot and it is placed in the ROM memory which means that it cannot be updated after manufacturing at all.
+The ROM E2E (End-to-End) tests validates the ROM features and can be used for regression tests.
+Each test is divided into components:
+ - **Opentitantool**, it is a Rust program that provide a common interface to interact with Opentitan. Opentitantool uses opentitanlib which provides the abstractions to communicate with Opentitan.
+ - **Host test** software, it is a Rust program built by bazel that actively interacts with the device software via UART, SPI or JTAG to perform the test and check whether it passed or not. To avoid code duplication the tests uses the `opentitanlib` to communicate with opentitan.
+ - **Device test** software, it is a C program built by bazel that run in the flash after the ROM and perform checking in the HW and/or memories to check the configurations performed by the ROM.
+ - **FPGA bitstream**, it is built by bazel which using vivado.
+ - **ROM Image**, it is a C program built by bazel which is the DUT (Device Under Test).
+ - **OTP image**, contains secure configurations read by the ROM to define workflows such as the lifecycle stage, so for some tests may be executed with several different OTP images to test all corner cases.
```

Comment:
```suggestion
 - **OTP image**, contains HW and SW configurations, some of which control execution paths in the ROM, such as, the lifecycle stage. As a result, we run some tests with several different OTP images to increase coverage of ROM execution paths.
```
maybe would be good to [link](https://opentitan.org/book/hw/ip/otp_ctrl/index.html#direct-access-memory-map) to the OTP `CREATOR_SW_CFG` table here?

Reply:
Done